README.md

```diff
@@ -68,10 +68,37 @@ $ yarn install
 # serve with hot reload at localhost:3000
 $ yarn dev
 
-# build for production and launch server
+# server with hot reload and server-side rendering at localhost:3000
+$ yarn dev:ssr
+
+# build for production
+# you will need a web server to host the client
 $ yarn build
-$ yarn start
 
-# generate static project
-$ yarn generate
+# build for production with server-side rendering and launch server
+$ yarn build:ssr
+$ yarn start:ssr
+
+```
+
+## Running standalone
+
+When using the SSR version of the client, you can run the client in standalone mode in order to save space.
+
+```bash
+# install dependencies
+$ yarn install
+
+# build for production with server-side rendering and launch server
+$ yarn build:ssr
+$ yarn start:ssr
```

Comment:
You can remove this I guess

Reply:
Why? It's different than the standalone mode.

Reply:
But those instructions are already there a few lines above. But maybe they're also needed in standalone?

Reply:
But there already are instructions for the non standalone mode above, isn't it?